CHANGELOG.md

```diff
@@ -39,6 +39,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
       - `webpack@4.24.0`
       - `webpack-command@0.4.2`
 
+### Fixed
```

Comment:
Update to  ` ###Changed` per keepachangelog.com

Reply:
Okay, I just went over the site again and they use both ### Changed and ### Fixed. Verbally we agreed to use Added and Changed. Discuss?